contract/src/gameAssetContract.js

```diff
@@ -23,8 +23,9 @@ const bagValueSize = amt => {
   return total;
 };
 
+/** @typedef {StandardTerms & { joinPrice: Amount<'nat'> }} GamePlacesTerms */
```

Comment:
**important**: Why constrain it to `nat` amounts?

`StandardTerms` is redundant here. The parameter to the `ZCF<...>` type is _custom_ terms.

Reply:
> StandardTerms is redundant here. The parameter to the ZCF<...> type is custom terms.

Agree/ack. This was added in an attempt to get `const terms = await E(zoe).getTerms(instance);` inside `test-contract.js` to not complain `Property 'brands' does not exist on type 'GamePlacesTerms'.`. 

Open to hints for getting `StandardTerms` recognized properly in the testing context.